websites/F/Free Web Novel/dist/metadata.json

```diff
@@ -0,0 +1,94 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "BigBoi",
+		"id": "853932304257908746"
+	},
+	"service": "Free Web Novel",
+	"description": {
+		"en": "We are offering free books online read! Read novel updated daily: light novel translations, web novel, chinese novel, english Novel, japanese novel and korean novel online."
```

Comment:
nationalities in English are spelled with an initial upper case. Also the casing of "novel" here is inconsistent. I believe you should be saying "novels" though

Reply:
Just took the description straight from the site, fixing it 👍